examples/Hermitian_Phasing.py

```diff
@@ -81,12 +81,9 @@
         data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
         data.ft("t2")
         data.ift("t2")
-        # {{{ this is not correct -- you want to use the zeroth order phasing
-        # routine to determine the zeroth order phase, since it allows for
-        # negative data this will flip the negative data to positive
-        ph0 = select_pathway(data, signal_pathway)["t2":0]
-        ph0 /= abs(ph0)
-        data /= ph0
+        # {{{ Zeroth Order Phase Correction
+        retval = zeroth_order_ph(select_pathway(data,signal_pathway))
+        data /= retval
         # }}}
```

Comment:
yes!  But we do want to show the diagnostic plots for this guy too.  (I would show the diagnostic plots in the Hermitian Phasing example but not in the correlation example)